### DISCUSSION TITLE
What noise should we include in our sims and should it come from deep CMB?

### DISCUSSION THREAD

**Discussion by evavagiakis** [deepskies/DeepSZSim#140] [Ideas]
We should discuss what kind of noise we want to include in a noise module (for first release, and then later), and should it come from existing deep skies code? @samueldmcdermott could you link to the noise scripts you currently have?

**Comment by samueldmcdermott** [comment]
https://github.com/deepskies/DeepCMBsim/blob/2bef647bc97c4b95f70c969063d653ae7c420674/deepcmbsim/noise.py#L29C82-L29C82
